### PR TITLE
Remove cgio_read_data , cgio_read_all_data, cgio_read_block_data from doc

### PR DESCRIPTION
**Pull request by MicK7**
Replace cgio_read_data by cgio_read_data_type in the documentation
Same with cgio_read_all_data and cgio_read_block_data. (CGNS-177)

### CODE REVIEW

File: CGNS_docs_current/cgio/dataio.html [hunk 1]
```diff
@@ -236,20 +260,20 @@ <h2>Function Descriptions</h2>
 
 <dt>&nbsp;<dd>
 <a name="write_data"></a>
-<dt><tt>cgio_write_data</tt>
-<dd>This function is similar to <a href="#read_data">cgio_read_data</a>,
+<dt><tt>cgio_write_data</tt>, <tt>cgio_write_data_type</tt>
+<dd>These function are similar to <a href="#read_data_type">cgio_read_data_type</a>,
```
Comment (brtnfld) [inline CGNS_docs_current/cgio/dataio.html]: Was correct, it should be "This" and "is"

**Review by brtnfld — APPROVED** [review]
Looks good, thanks!

File: CGNS_docs_current/cgio/dataio.html (hunk 1 above)
Comment (MicK7) [inline CGNS_docs_current/cgio/dataio.html]:
```suggestion
<dd>This function is similar to <a href="#read_data_type">cgio_read_data_type</a>,
```

File: CGNS_docs_current/cgio/dataio.html [hunk 2]
```diff
@@ -236,20 +260,20 @@ <h2>Function Descriptions</h2>
...
     but writes the data from memory to the node.
 
 <dt>&nbsp;<dd>
 <a name="write_all_data"></a>
-<dt><tt>cgio_write_all_data</tt>
-<dd>This function is similar to <a href="#read_all_data">cgio_read_all_data</a>,
+<dt><tt>cgio_write_all_data</tt>, <tt>cgio_write_all_data_type</tt>
+<dd>These function are similar to <a href="#read_all_data_type">cgio_read_all_data_type</a>,
```
Comment (MicK7) [inline CGNS_docs_current/cgio/dataio.html]:
```suggestion
<dd>This function is similar to <a href="#read_all_data_type">cgio_read_all_data_type</a>,
```

**Pull request merged**